src/runtime/components/Toaster.vue

```diff
@@ -120,18 +120,20 @@ function getOffset(index: number) {
       @click="toast.click && toast.click(toast)"
     />
 
-    <ToastViewport
-      :data-expanded="expanded"
-      :class="ui.viewport({ class: [props.class, props.ui?.viewport] })"
-      :style="{
-        '--scale-factor': '0.05',
-        '--translate-factor': position?.startsWith('top') ? '1px' : '-1px',
-        '--gap': position?.startsWith('top') ? '16px' : '-16px',
-        '--front-height': `${frontHeight}px`,
-        '--height': `${height}px`
-      }"
-      @mouseenter="hovered = true"
-      @mouseleave="hovered = false"
-    />
+    <TooltipPortal>
```

Comment:
Why use `TooltipPortal` and not the `Teleport` component from `vue`? 🤔

Reply:
When I use `Teleport`, I get an error 

<img width="1051" alt="Screenshot 2024-11-07 at 9 06 40 PM" src="https://github.com/user-attachments/assets/4e6c166d-cb85-4894-a21a-d6096a87c281">